scripts/update-schema.sh

```diff
@@ -14,12 +14,46 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-go run $GOPATH/src/github.com/openconfig/ygot/proto_generator/protogenerator.go \
-	-path=yang,yang/deps \
-	-output_dir=proto -compress_paths -generate_fakeroot -fakeroot_name=device \
+# Hack to ensure that if we are running on OS X with a homebrew installed
+# GNU sed then we can still run sed.
+runsed() {
+  if hash gsed 2>/dev/null; then
+    gsed "$@"
+  else
+    sed "$@"
+  fi
+}
+
+# A similar hack for readlink
+runreadlink() {
+  if hash greadlink 2>/dev/null; then
+    greadlink "$@"
+  else
+    readlink "$@"
+  fi
+}
+
+if [ -z $SRCDIR ]; then
+	THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+	SRC_DIR=`runreadlink -m ${THIS_DIR}/..`
+fi
+
+proto_generator \
+	-path=${SRC_DIR}/yang,${SRC_DIR}/yang/deps \
+	-output_dir=${SRC_DIR}/proto -compress_paths -generate_fakeroot -fakeroot_name=device \
 	-package_name=gribi_aft -exclude_modules=ietf-interfaces,openconfig-interfaces \
+	-base_import_path="proto" \
+	-go_package_base="github.com/openconfig/gribi/proto" \
+	-consistent_union_enum_names -typedef_enum_with_defmod \
+	${SRC_DIR}/yang/gribi-aft.yang
 	-base_import_path="github.com/openconfig/gribi/proto" yang/gribi-aft.yang
```

Comment:
Sorry didn't catch this PR in time. `-base_import_path` is specified twice here. Doesn't block the submission as I'm guessing the former is what's preferred to avoid references to the github repo's path, and the latter seems to be a syntax error that's skipped by the script. The proto generation flags LGTM.

Reply:
Thanks - missed this in the merge. I wonder why testing didn't cover this. Will ensure that there is some testing that covers this and send a new PR.